src/lib/components/SiteFooter/index.jsx

```diff
@@ -82,6 +91,21 @@ class SiteFooter extends React.Component {
     return mobileLinks;
   }
 
+  renderBusinessMarketingListItem() {
+    const { siteName, enterpriseMarketingLink } = this.props;
+    return enterpriseMarketingLink && (
```

Comment:
Personal preference, but I'm not a huge fan of && for flow control in this manner.

Reply:
What would be your preference in this case?

```
if (enterpriseMarketingLink) {
  return (<something/>);
}
return null;
```

Reply:
I'm a JS newb btw, so I'm always looking for new ways of doing things!

Reply:
@albemarle Also curious to hear your take on why you're not a huge fan of `&&` in this manner. Seems like it's a pretty common technique for flow control?